clash-lib/src/Clash/Driver/Types.hs

```diff
@@ -176,6 +180,7 @@ defClashOpts
   , opt_forceUndefined      = Nothing
   , opt_checkIDir           = True
   , opt_aggressiveXOpt      = False
+  , opt_inlineWFCacheLimit  = 10
```

Comment:
Add a `TODO` saying we should do a parameter sweep to find it's optimal (if it exists) value.